packages/api-client/src/APIClient.test.ts

```diff
@@ -281,10 +281,10 @@ describe('APIClient', () => {
       const client = new APIClient();
       const testError = new Error('Test rejection');
 
-      spyOn(client.api.auth, 'postLogout').and.returnValue(Promise.reject(testError));
-      spyOn(client, 'disconnect').and.returnValue();
-      spyOn(client['accessTokenStore'], 'delete').and.returnValue(Promise.resolve(undefined));
-      spyOn(client['logger'], 'warn').and.returnValue();
+      jest.spyOn(client.api.auth, 'postLogout').mockReturnValue(Promise.reject(testError));
+      jest.spyOn(client, 'disconnect').mockReturnValue();
+      jest.spyOn(client['accessTokenStore'], 'delete').mockReturnValue(Promise.resolve(undefined));
+      jest.spyOn(client['logger'], 'warn').mockReturnValue();
```

Comment:
maybe makes sense to destructure spyOn from jest just to have a few less characters?

Reply:
If you don't have strong opinion on this, I'd like to keep it this way. I don't think character count is a very useful metrics